lacework/resource_lacework_policy_compliance.go

```diff
@@ -0,0 +1,255 @@
+	)
+
+	log.Printf("[INFO] Reading Policy with guid %s\n", d.Id())
+	response, err := lacework.V2.Policy.Get(d.Id())
+	if err != nil {
+		return resourceNotFound(d, err)
+	}
+
+	d.SetId(response.Data.PolicyID)
+	d.Set("title", response.Data.Title)
+	d.Set("query_id", response.Data.QueryID)
+	d.Set("enabled", response.Data.Enabled)
+	d.Set("description", response.Data.Description)
+	d.Set("severity", response.Data.Severity)
+	d.Set("remediation", response.Data.Remediation)
+	d.Set("type", response.Data.PolicyType)
+	d.Set("owner", response.Data.Owner)
+	d.Set("updated_time", response.Data.LastUpdateTime)
+	d.Set("updated_by", response.Data.LastUpdateUser)
+	d.Set("computed_tags", strings.Join(response.Data.Tags, ","))
+
+	alerting := make(map[string]interface{})
+	alerting["enabled"] = response.Data.AlertEnabled
+	d.Set("alerting", alerting)
+
+	log.Printf("[INFO] Read Policy with guid %s\n", response.Data.PolicyID)
+	return nil
+}
+
+func resourceLaceworkPolicyComplianceUpdate(d *schema.ResourceData, meta interface{}) error {
+	var (
+		lacework = meta.(*api.Client)
+	)
+
+	if d.HasChange("policy_id_suffix") {
+		return errors.New("unable to change ID of an existing policy")
+	}
+
+	policyEnabled := d.Get("enabled").(bool)
+
+	policy := api.UpdatePolicy{
+		PolicyType:  policyType,
+		QueryID:     d.Get("query_id").(string),
+		Title:       d.Get("title").(string),
+		Enabled:     &policyEnabled,
+		Description: d.Get("description").(string),
+		Remediation: d.Get("remediation").(string),
+		Severity:    d.Get("severity").(string),
+		PolicyID:    d.Id(),
+		Tags:        castStringSlice(d.Get("tags").(*schema.Set).List()),
+	}
+
+	log.Printf("[INFO] Updating Policy with data:\n%+v\n", policy)
+	response, err := lacework.V2.Policy.Update(policy)
+	if err != nil {
+		return err
+	}
+
+	d.SetId(response.Data.PolicyID)
+	d.Set("owner", response.Data.Owner)
+	d.Set("updated_time", response.Data.LastUpdateTime)
+	d.Set("updated_by", response.Data.LastUpdateUser)
+	d.Set("computed_tags", strings.Join(response.Data.Tags, ","))
+
+	log.Printf("[INFO] Updated Policy with guid %s\n", response.Data.PolicyID)
+	return nil
+}
+
+func resourceLaceworkPolicyComplianceDelete(d *schema.ResourceData, meta interface{}) error {
+	lacework := meta.(*api.Client)
+
+	log.Printf("[INFO] Deleting Policy with guid %s\n", d.Id())
+	_, err := lacework.V2.Policy.Delete(d.Id())
+	if err != nil {
+		return err
+	}
+
+	log.Printf("[INFO] Deleted Policy with guid %s\n", d.Id())
+	return nil
+}
+
+func importLaceworkPolicyCompliance(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	lacework := meta.(*api.Client)
+
+	log.Printf("[INFO] Importing Lacework Policy with guid: %s\n", d.Id())
+
+	response, err := lacework.V2.Policy.Get(d.Id())
+	if err != nil {
+		return nil, fmt.Errorf(
+			"Unable to import Lacework resource. Policy with guid '%s' was not found",
+			d.Id(),
+		)
+	}
+	if response.Data.PolicyType != "Compliance" {
```

Comment:
For example, here we can do:
```suggestion
	if response.Data.PolicyType != api.PolicyTypeCompliance.String() {
```